### ISSUE TITLE
Hangs waiting for camera if laptop closed & running from external monitor

### ISSUE THREAD

**Issue by YourBrain** [wanderingstan/Lifeslice#17]
At least i think that is the problem.

**Comment by wanderingstan** [comment]
Yes, that's a problem! I suppose this means that there is no camera available, in which case it should do nothing. (Or perhaps give a one-time notification that webcam shots are not getting recorded.)
